Jenkinsfile

```diff
@@ -5,26 +5,27 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 ** Variables.
 */
 def serie = '21.10'
-def stableBranch = "${serie}.x"
-def devBranch = "dev-${serie}.x"
+def stableBranch = "21.10.x"
+def devBranch = "dev-21.10.x"
```

Comment:
I think you can revert those changes because the `serie` variable is defined above

Reply:
yup, I tried different things to make the pipeline run and those changes for where serie is replaced can definitely be reverted

Reply:
though there's no harm in leaving them as well, we have it hardcoded in the 22.04.x branch as well

Reply:
I think we can leave as it is. Just a question about best practises ^^

Reply:
for sure, I wouldn't mind doing it either way :)